requirements.txt

```diff
@@ -5,3 +5,5 @@ pyparsing==3.0.9
 PyYAML==6.0
 requests==2.28.1
 scipy==1.8.1
+dearpygui==1.8.0
+psutil==5.9.4

```

Comment:
`dearpygui` is the library used for drawing the UI
`psutil` is used to check the amount of available RAM and logical CPU cores